### PR TITLE
Document the Windows Subsystem for Linux

### PR DESCRIPTION
**Pull request by zhubonan**
Added guide for installing AiiDA on WSL in the documentation. I have tested the installation and there are a 2 failures (file modes with ssh transport and profile detetion) and 7 error out of 1051 tests. 
Nevertheless, I think overall it is OK to install on WSL if the user just want to try out on a windows machine especially when they don't have enough RAM to support a virtual machine.

This should allow close of #2256

### CODE REVIEW

**Comment by zhubonan** [comment]
@CasperWA

File: docs/source/install/details/os.rst [hunk 1]
```diff
@@ -95,4 +95,27 @@ If you want to manually start the RabbitMQ server you can use:
...
+-----------------------------------
+
+The guide for Ubuntu above can be followed but there are a few things to note:
+
+#. The `window native RabbitMQ <https://www.rabbitmq.com/install-windows.html>`_
+   should be installed and started as a service for WSL installation,
+   since the version in WSL Ubuntu 18.04 does not work properly.
+
+#. Linux services under WSL are not started automatically.
+   To start the Postgre SQL service, type the command below in the terminal::
```
Comment (giovannipizzi) [inline docs/source/install/details/os.rst]: PostgreSQL is a single word

File: docs/source/install/details/os.rst [hunk 2]
```diff
@@ -95,4 +95,27 @@ If you want to manually start the RabbitMQ server you can use:
 
     /etc/init.d/rabbitmq start
 
-For more information, or if you run into trouble, please refer to the RabbitMQ :ref:`troubleshooting section<installation_rabbitmq>`.
\ No newline at end of file
+For more information, or if you run into trouble, please refer to the RabbitMQ :ref:`troubleshooting section<installation_rabbitmq>`.
+
+
+Window Subsystem for Linux (Ubuntu)
```
Comment (giovannipizzi) [inline docs/source/install/details/os.rst]: Window->Windows

File: docs/source/install/details/os.rst [hunk 3]
```diff
@@ -95,4 +95,27 @@ If you want to manually start the RabbitMQ server you can use:
...
+-----------------------------------
+
+The guide for Ubuntu above can be followed but there are a few things to note:
+
+#. The `window native RabbitMQ <https://www.rabbitmq.com/install-windows.html>`_
```
Comment (giovannipizzi) [inline docs/source/install/details/os.rst]: Window->Windows

File: docs/source/install/details/os.rst [hunk 4]
```diff
@@ -95,4 +95,27 @@ If you want to manually start the RabbitMQ server you can use:
...
+#. The `windows native RabbitMQ <https://www.rabbitmq.com/install-windows.html>`_
+   should be installed and started as a service for WSL installation,
+   since the version in WSL Ubuntu 18.04 does not work properly.
+
+#. Linux services under WSL are not started automatically.
+   To start the PostgreSQL service, type the command below in the terminal::
+
+     sudo service postgresql start
+
+#. There is a known issue in WSL Ubuntu 18.04 where the timezone is not
```
Comment (ltalirz) [inline docs/source/install/details/os.rst]: perhaps link to issue?

File: docs/source/install/details/os.rst [hunk 5]
```diff
@@ -95,4 +95,27 @@ If you want to manually start the RabbitMQ server you can use:
...
+For more information, or if you run into trouble, please refer to the RabbitMQ :ref:`troubleshooting section<installation_rabbitmq>`.
+
+
+Windows Subsystem for Linux (Ubuntu)
+------------------------------------
+
+The guide for Ubuntu above can be followed but there are a few things to note:
+
+#. The `windows native RabbitMQ <https://www.rabbitmq.com/install-windows.html>`_
+   should be installed and started as a service for WSL installation,
```
Comment (ltalirz) [inline docs/source/install/details/os.rst]:
```suggestion
   should be installed and started
```

**Pull request merged**